content/actions/using-containerized-services/about-service-containers.md

```diff
@@ -83,11 +83,12 @@ You can map service containers ports to the Docker host using the `ports` keywor
 |------------------|--------------|
 | `8080:80` |	Maps TCP port 80 in the container to port 8080 on the Docker host. |
 | `8080:80/udp` |	Maps UDP port 80 in the container to port 8080 on the Docker host. |
-| `8080/udp`	| Map a randomly chosen UDP port in the container to UDP port 8080 on the Docker host. |
+| `8080/udp`	| Maps a randomly chosen UDP port in the container to UDP port 8080 on the Docker host. |
+| `8080` | Maps TCP port 8080 of the container to random free port on the Docker host. |
 
 When you map ports using the `ports` keyword, {% data variables.product.prodname_dotcom %} uses the `--publish` command to publish the container’s ports to the Docker host. For more information, see "[Docker container networking](https://docs.docker.com/config/containers/container-networking/)" in the Docker documentation.
 
-When you specify the Docker host port but not the container port, the container port is randomly assigned to a free port. {% data variables.product.prodname_dotcom %} sets the assigned container port in the service container context. For example, for a `redis` service container, if you configured the Docker host port 5432, you can access the corresponding container port using the `job.services.redis.ports[5432]` context. For more information, see "[Contexts](/actions/learn-github-actions/contexts#job-context)."
+When you specify the Docker container port but not the host port, the host port is randomly assigned to a free port. {% data variables.product.prodname_dotcom %} sets the assigned host port in the service container context. For example, for a `redis` service container, if you configured the Docker container port 5432, you can access the corresponding host port using the `job.services.redis.ports[5432]` context. For more information, see "[Contexts](/actions/learn-github-actions/contexts#job-context)."
```

Comment:
👋 Thanks for this PR, @mindtraveller, but IIUC, these changes aren't correct. Only specifying one value for `port` will set the Docker **host** port to that value, and when the container is created, the port **in the container** will be randomly generated. If the host port was randomly generated, then other processes on the runner wouldn't know the port to be able to access the container's service.

Reply:
🤔 I admit that I'm now getting confused 😅, but I'm certainly no Docker expert. I'll try and get one of our Actions engineers to confirm 🙂